renderer/components/Form/OpenDialogInput.js

```diff
@@ -19,14 +19,15 @@ const InnerInput = styled(BasicInput)`
 `
 
 const OpenDialogInput = props => {
-  const { mode, ...rest } = props
+  const { mode, label, ...rest } = props
   const [spaceProps, otherProps] = extractSpaceProps(rest)
   return (
     <WithOpenDialog
       render={({ openDialog }) => (
-        <Flex alignItems="center" {...spaceProps}>
-          <InnerInput width={1} {...otherProps} />
+        <Flex alignItems="flex" {...spaceProps}>
```

Comment:
Is `flex` a valid option for `alignItems`?

Reply:
lol, good catch. that should go completely